src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -290,7 +290,7 @@ class DbtCoreOperation(ShellOperation):
 
     _block_type_name = "dbt Core Operation"
     _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/9a001902ed43a84c6c96d23b24622e19/dbt-bit_tm.png?h=250"  # noqa
-    _documentation_url = "https://prefecthq.github.io/prefect-dbt/cli/commands/#prefect_dbt.cli.commands.DbtCoreOperation"  # noqa
+    _documentation_url = "https://docs.prefect.io/integrations/prefect-dbt"  # noqa
```

Comment:
are we keeping netlify? if so can we link to the actual api refs instead of the top level docs for the integration?

Reply:
Good question. The plan is not to keep the Netlify SDK docs long term. Mintlify should have an SDK builder available soon and the plan is to switch to that. Once those docs are up, linking directly to the relevant SDK docs could be a good plan.